.evergreen/scripts/env-var-utils.sh

```diff
@@ -1,38 +1,12 @@
 #!/usr/bin/env bash
 
-# Usage:
-#   to_absolute "./relative/path/to/file"
-#   to_absolute "./relative/path/to/dir"
-#   to_absolute "/absolute/path/to/file"
-#   to_absolute "/absolute/path/to/dir"
-#
-# Uses dirname, basename, and pwd only. Unix-style paths only!
-to_absolute() (
-  declare path="${1:?"to_absolute requires a path to convert"}"
-
-  declare dir_part base_part
-  dir_part="$(dirname "${path}")" || return
-  base_part="$(basename "${path}")" || return
-
-  declare res
-
-  if [[ -d "${path}" ]]; then
-    cd "${path}" || return
-    res="$(pwd)" || return
-  else
-    res="$(to_absolute "${dir_part}")" || return
-    [[ "${res: -1:1}" == "/" ]] || res+="/" || return
-    [[ "${base_part}" == "~" ]] || res+="${base_part}" || return
-  fi
-
-  printf "%s" "${res:?}"
-)
+. "$(dirname "${BASH_SOURCE[0]}")/../../tools/init.sh"
```

Comment:
Suggest reverting changes to this file, as linking `env-var-utils.sh` -> `init.sh` indirectly introduces a hidden `init.sh` depencency in all scripts that currently include `env-var-utils.sh`. See other comment expressing concerns regarding "elephant on a flea" and "viral" behavior of `init.sh`.